docs/UserGuide.md

```diff
@@ -503,21 +509,22 @@ use the `edit` command to input the accurate details._
 >
 > At least one of the optional fields must be provided.
 
-**Examples:** 
+**Examples:**
 
-1. You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
+**Step 1.** You received an email from Microsoft to interview for a Frontend Developer role instead, which is different from the Software Engineer role you applied for.
 To edit the company's information, type `edit 2 s/PI r/Frontend Developer`
 
 ![EditDemo1.png](images/edit-command/EditDemo1.png)
-2. The status and role of the 2nd company in the list is edited to be Pending Interview and Frontend Developer respectively.
+
+**Step 2.** The status and role of the 2nd company in the list is edited to be Pending Interview and Frontend Developer respectively.
 
 ![EditDemo2.png](images/edit-command/EditDemo2.png)
 
-3. Go ahead and edit more companies on your own! 
+**Step 3.** Go ahead and edit more companies on your own! 
```

Comment:
just wondering if this sounds bit funny, like asking them to edit more, but if their input doesn't have any errors, technically they don't have to edit? 

Maybe just say "Some other examples you may try to familiarise with the edit command?" 

Something along the line?